src/app/modules/sample-providers/routes/submissionSuccess.scss

```diff
@@ -0,0 +1,52 @@
+:host {
+  width: 100%;
+  height: 100%;
+  position: fixed;
```

Comment:
It seems suspicious to me when we start using things like `position: fixed`. If we removed the padding on the `body` and the `container` class we wouldn't have to rely on negative margins. Which, in return would also make it to where we wouldn't need to use `position: fixed`. There may be reasons that this is needed that I'm missing. I'd love to get your thoughts on this :)

![image](https://user-images.githubusercontent.com/5473222/134713185-98977d56-91ae-4c21-b4bd-ef537b9700ef.png)

![image](https://user-images.githubusercontent.com/5473222/134713435-8925c785-91dd-41b6-8688-dd6222455d7d.png)